packages/studio-plugin/src/types/StudioTailwindConfig.ts

```diff
@@ -0,0 +1,19 @@
+import type { Config } from "tailwindcss";
+
+/**
+ * The subset of a tailwind Config supported by Studio.
+ */
+export type StudioTailwindConfig = Omit<Config, "theme" | "content"> & {
+  content: string[];
+  theme: {
+    extend: StudioTailwindTheme;
```

Comment:
Should `theme` and `extend` be optional properties?

Reply:
will update!